spec/unit/pops/types/p_object_type_spec.rb

```diff
@@ -687,6 +675,35 @@ def parse_object(name, body_string)
       expect(eval_and_collect_notices(code)).to eql(['true'])
     end
 
+    it 'a named object is added to the loader irrespective of alias declaration' do
+      code = <<-CODE
+      $x = Object[{ name => 'MyFirstObject' }]
+      notice($x == MyFirstObject)
+      CODE
```

Comment:
If that is wrong, then it is also wrong that this bind the object to two names.
```
type Foo = Object[{name => 'Bar'}]
```
I thought we agreed that it should?